packages/protocol/contracts/L1/hooks/AssignmentHook.sol

```diff
@@ -41,6 +41,11 @@ contract AssignmentHook is EssentialContract, IHook {
     /// the prover's job to make sure it can get paid within this limit.
     uint256 public constant MAX_GAS_PAYING_PROVER = 50_000;
 
+    /// @notice bytes32 representation of the string "taiko_token".
+    bytes32 private constant TKO = bytes32("taiko_token");
+    /// @notice Constant string "PROVER_ASSIGNMENT".
+    string private constant PROVER_ASSIGNMENT = "PROVER_ASSIGNMENT";
```

Comment:
how about we also convert this to bytes32?

Reply:
We can - no issue, but we canot do with "SIGNAL", (bc. of tests)

Reply:
Oh, then keep them as-is not to break upgradability then.